workspace/src/app/views/shared/modals/CreateArtefactModal/ArtefactForms/ParameterAutoCompletion.tsx

```diff
@@ -168,13 +173,33 @@ export const ParameterAutoCompletion = ({
 
     const itemValue = (value: StringOrReifiedValue) => (typeof value === "string" ? value : value.value);
 
+    const handleSearch = React.useCallback(
+        (input: string) => {
+            setSearchQuery(input);
+            setLimit(AUTOCOMPLETION_LIMIT)
+            setShouldLoadMoreResults(true)
+            return handleAutoCompleteInput(input, autoCompletion);
+        },
+        [limit]
+    );
+
+    const loadMoreResults = async () => {
+        if(shouldLoadMoreResults){
+            const newLimit = limit + AUTOCOMPLETION_LIMIT;
+            const results = (await handleAutoCompleteInput(searchQuery, autoCompletion, newLimit)) ?? [];
+            setLimit(newLimit);
+            setShouldLoadMoreResults(results.length >= newLimit)
+            return results.slice(limit + 1)
```

Comment:
It's because of a line, I added months ago where I unshift an `Empty Path` into the results. Can't remember why 🙃, can you. I have removed. Hopefully nothing breaks